src/main/scala/cromwell/util/google/GoogleCredentialFactory.scala

```diff
@@ -40,6 +41,13 @@ object GoogleCredentialFactory {
     GoogleClientSecrets.load(jsonFactory, secretStream)
   }
 
+  private def validateCredentials(credential: Credential) = {
+    Try(credential.refreshToken()) match {
+      case Failure(ex) => throw new Throwable(s"Google credentials are invalid: ${ex.getMessage}")
```

Comment:
Obviously I'm a little late with this observation, but it might be better to give the original `ex` as the `cause` of the new `Throwable` to preserve the original stack trace.